db/seeds/team/torrocus.rb

```diff
@@ -56,17 +56,17 @@
 
 TechnologicalSkill.where(person_id: torrocus.id).delete_all
 
-# minitest
 # testcase
 {
   'ruby' => { experience_rating: 91 },
-  'ruby-on-rails' => { experience_rating: 81 },
+  'ruby-on-rails' => { experience_rating: 82 },
   'sinatra' => { experience_rating: 60 },
-  'jekyll' => { experience_rating: 82 },
+  'jekyll' => { experience_rating: 83 },
   'jruby' => { experience_rating: 50 },
   'truffleruby' => { experience_rating: 40 },
 
   'rspec' => { experience_rating: 72 },
+  'minitest' => { experience_rating: 50 },
```

Comment:
Maybe a higher value?

Reply:
This value is ok for me now. I will keep an eye on it.